frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.tsx

```diff
@@ -587,7 +586,7 @@ export function FieldValuesWidgetInner({
         maw={maxWidth ?? undefined}
         miw={minWidth ?? undefined}
       >
-        {isListMode && isLoading ? (
+        {isListMode && multi && isLoading ? (
```

Comment:
Can we avoid having `isLoading` logic (i.e. rendering loading state) in 2 places?

2 reasons:
- it's confusing that this is handled on 2 levels, sometimes here, sometimes there
- loading states differ visually